Lesson3/Program.cs

```diff
@@ -1,12 +1,42 @@
-﻿using System;
+﻿using Project.ConsoleApp.Parser;
+using Project.Interface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace Lesson3
+namespace Project.ConsoleApp
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] clArguments)
         {
-            Console.WriteLine("New lesson 3 command line.");
-        }
+            if (clArguments.Any(a => String.Compare(a, "-json", true) == 0) || clArguments.Any(a => String.Compare(a, "-xml", true) == 0))
+            {
+                IFileHandling ifhApp = new ConsoleAppParser(clArguments);
+
+                //create a list then order it so nulls are last in the list
+                List<string> parsedData = ifhApp.GetParsedData(GetFilePath(clArguments, "-json"))
+                    .Concat(ifhApp.GetParsedData(GetFilePath(clArguments, "-xml")))
+                    .OrderBy(fh => fh)
+                    .ToList();
+
+                    parsedData.ForEach(vi => Console.WriteLine("{0}", vi));
+            }
+            else
+            {
+                throw new ArgumentException("Invalid arguments.");
+            }
+
+            string GetFilePath(string[] arguments, string argumentNameValue)
```

Comment:
`Observation`: If you added a check for file exists in this method, you could optionally provide a useful test that verifies the file and throws an exception if it doesn't exist.